ts/screens/messages/PaginatedMessagesHomeScreen.tsx

```diff
@@ -0,0 +1,171 @@
+/**
+ * A screen that contains all the Tabs related to messages.
+ */
+import * as React from "react";
+import { Animated } from "react-native";
+import { NavigationEventSubscription } from "react-navigation";
+import { NavigationStackScreenProps } from "react-navigation-stack";
+import { connect } from "react-redux";
+import { Millisecond } from "italia-ts-commons/lib/units";
+
+import MessagesSearch from "../../components/messages/paginated/MessagesSearch";
+import { ContextualHelpPropsMarkdown } from "../../components/screens/BaseScreenComponent";
+import { ScreenContentHeader } from "../../components/screens/ScreenContentHeader";
+import TopScreenComponent from "../../components/screens/TopScreenComponent";
+import { MIN_CHARACTER_SEARCH_TEXT } from "../../components/search/SearchButton";
+import { SearchNoResultMessage } from "../../components/search/SearchNoResultMessage";
+import SectionStatusComponent from "../../components/SectionStatus";
+import I18n from "../../i18n";
+import { loadNextPageMessages } from "../../store/actions/messages";
+import { Dispatch } from "../../store/actions/types";
+import {
+  isSearchMessagesEnabledSelector,
+  searchTextSelector
+} from "../../store/reducers/search";
+import { GlobalState } from "../../store/reducers/types";
+import customVariables from "../../theme/variables";
+import { HEADER_HEIGHT, MESSAGE_ICON_HEIGHT } from "../../utils/constants";
+import { setStatusBarColorAndBackground } from "../../utils/statusBar";
+import { sectionStatusSelector } from "../../store/reducers/backendStatus";
+import { setAccessibilityFocus } from "../../utils/accessibility";
+import { allMessagesSelector } from "../../store/reducers/entities/messages/allPaginated";
+import { pageSize } from "../../config";
+import HomeTabs from "../../components/messages/HomeTabs";
+import MessageList from "../../components/messages/paginated/MessageList";
+
+type Props = NavigationStackScreenProps &
+  ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+type State = {
+  currentTab: number;
+};
+
+const AnimatedScreenContentHeader =
+  Animated.createAnimatedComponent(ScreenContentHeader);
+
+const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
+  title: "messages.contextualHelpTitle",
+  body: "messages.contextualHelpContent"
+};
+
+/**
+ * A screen that contains all the Tabs related to messages.
+ */
+class MessagesHomeScreen extends React.PureComponent<Props, State> {
+  navListener?: NavigationEventSubscription;
+
+  animatedTabScrollPositions: ReadonlyArray<Animated.Value> = [
+    new Animated.Value(0),
+    new Animated.Value(0),
+    new Animated.Value(0)
```

Comment:
```suggestion
```
Since at the moment we have only a tab could be removed position 1-2?

Reply:
Makes sense, removed in 97a8cce3870a58f07649a9143be6e79a6d940bc5